pybamm/models/submodels/electrolyte_conductivity/surface_potential_form/full_surface_form_conductivity.py

```diff
@@ -136,26 +130,30 @@ def _get_conductivities(self, variables):
         return conductivity, sigma_eff
 
     def set_initial_conditions(self, variables):
-        if self.domain == "Separator":
+        Domain = self.domain.capitalize()
+
+        if self.domain == "separator":
             return
```

Comment:
Is this equivalent to `return None` (used below)? If so, should we keep it consistent (even if the output is the same)?

Reply:
I looked it up and there is a stylistic difference. I will try to keep it consistent https://stackoverflow.com/a/15300671/10816761